src/arch/riscv/head.S

```diff
@@ -17,8 +17,8 @@
 .extern trap_entry
 
 /*
- * When SMP is enabled, the elfloader passes the hart ID in a4
- * and logical core ID in a5.
+ * When SMP is enabled, the elfloader passes the hart ID in a6
```

Comment:
You mention one additional is added; why do we skip registers from a4,a5 to a6, a7. Also, the commit title is kind of misleading since this is an omission when changing the elfloader I think.

Reply:
Sorry it was two. `(paddr_t) dtb` and  `dtb_size`